AppLovin MAX Demo App - Kotlin/app/build.gradle

```diff
@@ -48,7 +58,12 @@ dependencies {
     implementation 'com.android.installreferrer:installreferrer:2.2'
 
     implementation 'com.google.android.gms:play-services-ads-identifier:17.1.0'
-    
+
+    implementation("androidx.compose.runtime:runtime:1.1.0")
+    implementation 'androidx.activity:activity-compose:1.3.0'
+    implementation "androidx.compose.material:material:1.3.0"
+    implementation 'androidx.compose.foundation:foundation:1.3.0'
```

Comment:
Thanks for slimming down the dependencies 👍 One small nit is that the `() " '` usages are inconsistent